packages/gestalt/src/Masonry.js

```diff
@@ -375,12 +378,19 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
     idx,
     position,
   ) => {
-    const { Item, scrollContainer, virtualize, virtualBoundsTop, virtualBoundsBottom } = this.props;
+    const {
+      Item,
+      scrollContainer,
+      virtualize,
+      virtualBoundsTop,
+      virtualBoundsBottom,
+      virtualBufferFactor,
+    } = this.props;
     const { top, left, width, height } = position;
 
     let isVisible;
-    if (scrollContainer) {
-      const virtualBuffer = this.containerHeight * VIRTUAL_BUFFER_FACTOR;
+    if (scrollContainer && virtualBufferFactor) {
```

Comment:
Is this check necessary, given that we're providing a default value in defaultProps? Or is this just to appease Flow?

Reply:
@dangerismycat just to appease flow